.cluster/cluster-conf.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: eksctl.io/v1alpha5
```

Comment:
why do we need this file?

Reply:
This file is not needed to make homework done if I use kind or minikube, but I need it for starting and stopping AWS Kubernetes cluster.